brokers.py

```diff
@@ -33,13 +33,15 @@ def isFileForBroker(cls, filename):
 from interactive_brokers import InteractiveBrokers
 from tdameritrade import TDAmeritrade
 from vanguard import Vanguard
+from schwab import Schwab
 
 
 BROKERS = {
     'amtd': TDAmeritrade,
     'ib': InteractiveBrokers,
     'tdameritrade': TDAmeritrade,
     'vanguard': Vanguard,
+    'schwab' : Schwab
```

Comment:
Please terminate this line with a comma as above to make it easier to add / remove brokers in the future, without modifying other lines around them.